scripts/release/steps/show-instructions-after-npm-publish.js

```diff
@@ -26,8 +27,12 @@ export function getReleaseUrl(version, previousVersion) {
       body: `🔗 [Release note](https://prettier.io/${blogPostInfo.path})`,
     });
   }
-  body = encodeURIComponent(body);
-  return `https://github.com/prettier/prettier/releases/new?tag=${version}&title=${version}&body=${body}`;
+  const parameters = new URLSearchParams({
```

Comment:
:+1:

Reply:
Actually I didn't mean to make this change here. Changed only because `encodeURIComponent` don't escaped `)`, but `URLSearchParams` do, maybe it's bug?

Reply:
https://stackoverflow.com/questions/59889140/different-output-from-encodeuricomponent-vs-urlsearchparams